code/game/objects/items/food/egg.dm

```diff
@@ -85,13 +85,17 @@
 /obj/item/food/egg/yellow
 	icon_state = "egg-yellow"
 
+/obj/item/food/egg/MakeGrillable()
```

Comment:
this should work like how pancakes work. Putting an egg ontop of a grill is not how you make a fried egg.